pkg/datasources/datasource_user.go

```diff
@@ -0,0 +1,76 @@
+package datasources
+
+import (
+	"context"
+
+	"github.com/MaterializeInc/terraform-provider-materialize/pkg/frontegg"
+	"github.com/MaterializeInc/terraform-provider-materialize/pkg/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func User() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: userDataSourceRead,
+
+		Description: `The user data source allows you to retrieve information about a user in your Materialize organization.`,
+
+		Schema: map[string]*schema.Schema{
+			"email": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The email address of the user to retrieve.",
+			},
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The unique (UUID) identifier of the user.",
+			},
+			"verified": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Whether the user's email address has been verified.",
+			},
+			"metadata": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Additional metadata associated with the user.",
+			},
```

Comment:
This field is used internally by Materialize Cloud. It's not strictly speaking sensitive, but it's also not something end users should be looking at. I'd just remove it, if that works!

Reply:
Good points! Just updated the PR to remove both the `metadata` and the `auth_provider` attributes.